backup/test_backup.py

```diff
@@ -207,3 +208,24 @@ def test_warning(directory, node_factory):
     assert(l1.daemon.is_in_log(
         r'The `--backup-destination` option is deprecated and will be removed in future versions of the backup plugin.'
     ))
+
+
+def test_rewrite():
+    tests = [
+        (
+            r'UPDATE outputs SET status=123, reserved_til=1891733WHERE prev_out_tx=1 AND prev_out_index=2',
+            r'UPDATE outputs SET status=123, reserved_til=1891733 WHERE prev_out_tx=1 AND prev_out_index=2',
+        ),
```

Comment:
Nit: you could also test for the second statement: `... peer_id=12345WHERE channels.id...`